### PR TITLE
feat: [Import Assistant] Page transformations

### PR DESCRIPTION
**Pull request by arumsey**
## Description

Add an assistant function that understands how to construct a prompt that will generate a transform function. The transform function should take the main element as input and the function can make manipulations to the main element DOM. The function does not need to return anything since DOM changes will be applied immediately.

- add page assistant command
- fix copying of templates

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/assistant/assistant-helper.js [hunk 1]
```diff
@@ -136,14 +136,23 @@ const runBlockAssistant = async ({url, name, prompt, outputPath = DEFAULT_IMPORT
...
   const builder = await getBuilder(url, {useExisting:  true, outputPath});
-  const manifest = await builder.addCells(name, prompt);
+  const manifest = await builder.addCellParser(name, prompt);
   writeManifestFiles(manifest, outputPath);
   console.log(chalk.green(`${name} block parser generated successfully in ${getDurationText(startTime)}`));
 };
 
+const runPageAssistant = async ({url, name, prompt, outputPath = DEFAULT_IMPORTER_PATH}) => {
+  const startTime = Date.now();
+  const builder = await getBuilder(url, {useExisting:  true, outputPath});
```
Comment (atopper) [inline src/assistant/assistant-helper.js:146]: Picky - double space in there.  (Maybe fix if there are any more changes going in.)

File: src/assistant/assistant-server.js [hunk 2]
```diff
@@ -12,6 +12,11 @@
 
 import path from 'path';
 import {copyFiles} from '../utils/fileUtils.js';
+import {findUpSync} from 'find-up';
+import { fileURLToPath } from 'url';
```
Comment (atopper) [inline src/assistant/assistant-server.js:16]: Style: padding "{ }" with spaces.  I thought lint enforced it, but there are 2 styles here (line 14-16).

Comment (arumsey) [reply src/assistant/assistant-server.js:16]: There is no linting on this project yet.

**Review by atopper — APPROVED** [review]
👍

**Pull request merged**